docs/en/latest/deployments/ack.md

```diff
@@ -36,11 +36,10 @@ This document explains how to install Ingress APISIX on [ali-cloud ACK](https://
 As the data plane of apisix-ingress-controller, [Apache APISIX](http://apisix.apache.org/) can be deployed at the same time using Helm chart.
 
 ```shell
-cd /path/to/apisix-helm-chart
-helm repo add bitnami https://charts.bitnami.com/bitnami
```

Comment:
We shouldn't remove this line. APISIX needs bitnami/etcd

Reply:
Ah I now understand, the `cd /path/to/apisix-helm-chart` step is because we are expecting the user to have a local copy of the chart, thus the `helm install apisix charts/apisix` is referring to a relative path on the local file system not a repo. This was not immediately obvious because the chart is not present in this repo.

I think it would be best if the install documentation used a helm repo, but I see you are considering moving hosting to github.

Can I suggest an explicit step to clone the helm chart from github until the repo hosting is resolved?

Reply:
We recommend people to use the online repo, so I think `helm repo add apisix https://charts.apiseven.com` is OK.